plugin.cpp

```diff
@@ -68,8 +68,9 @@ static const char *default_config = QUOTE({
         },
         "subscription" : {
                 "description" : "OPC UA Variables or Objects to observe for data changes",
-                "type" : "JSON",
-                "default" : "{ \"subscriptions\" : [  \"ns=3;i=1001\", \"ns=3;i=1002\" ] }",
+                "type" : "list",
```

Comment:
Add a listName property of subscription to keep the JSON format the same. See
https://docs.google.com/document/d/1_Zqn2B32wyPzuJCL-8x0qnVM1383xloHuZXi5AXMOzw/edit?tab=t.0#heading=h.4xpy2upq0a9a

Reply:
I don’t completely understand the rationale behind backward compatibility. Could you clarify the context and scope for me? Here are some specific questions I have:

**Does it mean that a new version of a plugin should work with the old core?**  
In other words, if someone has an older version of the core system, can they still use the latest plugin without issues?

**Or does it mean that the new core should support older versions of plugins?**  
If someone upgrades to the latest core, will their existing plugins still function properly?

**Or should existing users be able to upgrade to the new version of the plugin that is available on the new core version?**  
We want to ensure that customers can easily transition to new versions without losing functionality.

**Or is it about existing internal API compatibility?**  
If there are changes to the internal APIs, will that impact current plugins or other components that rely on those APIs?

I also want to note that in the proposed implementation, we are changing the plugin information, which is a breaking change.

Any insights you could provide would be greatly appreciated!